src/os/unix/media.cpp

```diff
@@ -10,11 +10,12 @@
 * Linux-specific implementation of media.h
 */
 
+// FIXME-QA(Krey): Why are we providing linux header files within the repository?
 #include "../media.h"
 #include "../file.h"
 #include "../utf8conv.h"
 
-#include <magic.h>
+#include "linux/magic.h"
```

Comment:
Note that on linux the `magic.h` file is saved in `/usr/include/linux/magic.h` relevant package provided.

Recommends using:
- https://pkgs.org/ | Database with linux distribution package manifests
- Command `dlocate` | On debian-based OS it can be used to find the files i.e `dlocate magic.h`

Reply:
Looks to me like linux/magic.h is unrelated to magic.h:
linux/magic.h defines kernel-related magic numbers, e.g., for the different filesystems.
magic.h defines the interface to libmagic, which is what PasswordSafe uses.

Reply:
@ronys noted, updated OP with TODO list to resolve this prior to merging assuming this being a snapshot from https://github.com/kreyren/forkless-pwsafe waiting for blocker resolution.

Sufficient?